lib/screens/explore_screens/vouch_for_a_member/interactor/viewmodel/vouch_for_a_member_state.dart

```diff
@@ -16,28 +18,33 @@ class VouchForAMemberState extends Equatable {
   @override
   List<Object?> get props => [
         pageState,
+        errorMessage,
         selectedMember,
         noShowUsers,
         pageCommand,
       ];
 
   VouchForAMemberState copyWith({
     PageState? pageState,
+    String? errorMessage,
     MemberModel? selectedMember,
     List<String>? noShowUsers,
     PageCommand? pageCommand,
   }) {
     return VouchForAMemberState(
       pageState: pageState ?? this.pageState,
+      errorMessage: errorMessage,
       selectedMember: selectedMember ?? this.selectedMember,
       noShowUsers: noShowUsers ?? this.noShowUsers,
       pageCommand: pageCommand,
     );
   }
 
-  factory VouchForAMemberState.initial() {
+  factory VouchForAMemberState.initial(List<MemberModel> alreadyVouched) {
     final List<String> noShowUsers = [settingsStorage.accountName];
 
+    noShowUsers.addAll(alreadyVouched.map((e) => e.account));
```

Comment:
what happens when you add a new person, then you go and try the same person without leaving the page. 
Does it show?